app/queries/mentoring_query.rb

```diff
@@ -13,7 +13,8 @@ def call
       .select(
         'array_agg(users.name) as mentees',
         'reviewers_users.name as mentors',
-        :city
+        :city,
+        "count('x') OVER (PARTITION BY offices.city) as city_count"
```

Comment:
what is this part of the query doing exactly?

Reply:
Only to count how many times a city appears, so it can be used to span table row.